src/qmflows/backports.py

```diff
@@ -61,34 +60,23 @@ def __exit__(self, exc_type, exc_value, traceback):
         pass
 
 
-try:
-    from contextlib import nullcontext  # type:ignore
-except ImportError:  # nullcontext was added in python 3.7
+# nullcontext was added in python 3.7
+if sys.version_info >= (3, 7):
+    from contextlib import nullcontext
+else:
     nullcontext = _NullContextBackup
-    nullcontext.__name__ = 'nullcontext'  # type: ignore
-
-
-try:  # Plan A: literal was added in Python 3.8
-    from typing import Literal  # type:ignore
-except ImportError:
-    try:  # Plan B: literal was previously available in a third party package
-        from typing_extensions import Literal  # type:ignore
-
-    except ImportError:
-        # Plan C; Literal.__getitem__ will now simply return the type
-        # of the passed object; for example: Literal[True] == bool
-        _LiteralBackup.__name__ = 'Literal'
-        Literal = _LiteralBackup()
-
-
-try:  # Plan A: Final was added in Python 3.8
-    from typing import Final  # type:ignore
-except ImportError:
-    try:  # Plan B: Final was previously available in a third party package
-        from typing_extensions import Final  # type:ignore
-
-    except ImportError:
-        # Plan C; Final.__getitem__ will now simply return the type
-        # of the passed object; for example: Final[bool] == bool
-        _FinalBackup.__name__ = 'Final'
-        Final = _FinalBackup()
+    nullcontext.__name__ = nullcontext.__qualname__ = 'nullcontext'
+
+
+# Literal and Literal were added to Python in 3.8;
```

Comment:
Just a typo in the comment Final and Literal

Reply:
Ah right, I'll get to it in a bit (as well as the, now broken, ``test/test_backports.py`` test).

Annoyingly enough it also seems like one of the ``test/test_using_plams/test_cp2k_opt.py`` tests broke since a recent PLAMS update.

Reply:
@BvB93 I have a useful (annoying) bot that copies the CI failure message into the PR. Also, I have made the GitHub actions  status mandatory before merging into the master but if you don't like those rules you can skip them if you have a good reason to do so ;)